### PR TITLE
Rework of #9145 - Rules id 91601 and 91602 are duplicated

### PR DESCRIPTION
**Pull request by Spades0**
## Description
|Related issue|Related PR|Closes Issue|
|---|---|---|
| [#9145](https://github.com/wazuh/wazuh/issues/9145) | #9146    |[#11211 ](https://github.com/wazuh/wazuh/issues/11211)|

## Changes & Comments
The aim of this PR is:

- added powershell.ini(contents are commented out though so as to pass jenkin tests because of json decoders.)
- Add Copyright header
- Add powershell.ini file and removed sample logs from the rules. 
- To ensure the rules and decoders from issue [#9145](https://github.com/wazuh/wazuh/issues/9145) work correctly.
- To fix issues regarding indentation, syntax, rule granularity, spellings and ensure the ruleset quality is standard.

### Syntax

- [x] 1.a Rule tags order must be compliant.
- [x] 1.b Decoder tags order must be compliant.
- [x] 1.c XML blocks must as compact as possible.
- [x] 1.d Only one empty line between rule/decoder and the next rule/decoder.
- [x] 1.e Decoder extracted fields names must use '_' whether a space is needed.
- [x] 1.f Decoder name must use '-' whether a space is needed.
- [x] 2.d XML and ini files must use correct indentation
 
### Grammar

- [x] 2.a Grammar quality.
- [x] 2.b Similar phrases must keep tenses and expressions.
- [x] 2.c Grammar basic rules like capitalization, punctuation marks, sentence construction, and others.

### Semantic

- [x] 3.a New decoder, rule, or test are written in the correct file and grouped correctly inside the file.
- [x] 3.b Group similar rules under same ID's group.
- [x] 3.c: 
   - [x] 3.c1 Find and reuse group name before creating a new one.
   - [x] 3.c2 Include a new group definition in a PR comment.
   - [x] 3.c3 Any group name must use '_' whether it does need a space char.
   - [x] 3.c4 The groups inside a tag must be sorted in alphabetical order.
   - [x] 3.d Rule level should be compliant with the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html).

### Unit testing

- [x] 4.a A metadata block at the beginning of the .ini file describing software, version, and logs source.
- [x] 4.b Each new rule must have at least one test entry in the correct .ini file.
- [x] 4.c Runtest.py must pass and must include the results in raw format here.
<details>
  <statement>Unit Tests</statement>

  ```
Restarting wazuh-manager...
- [ File = ./tests/samba.ini ] ---------
....

- [ File = ./tests/fortigate.ini ] ---------
........................................

- [ File = ./tests/apparmor.ini ] ---------
.....

- [ File = ./tests/pam.ini ] ---------
.....

- [ File = ./tests/audit_lateral.ini ] ---------
......

- [ File = ./tests/systemd.ini ] ---------
..

- [ File = ./tests/arbor.ini ] ---------
..

- [ File = ./tests/dovecot.ini ] ---------
...............

- [ File = ./tests/mailscanner.ini ] ---------
.

- [ File = ./tests/ossec.ini ] ---------
.....

- [ File = ./tests/vuln_detector.ini ] ---------
..

- [ File = ./tests/squid_rules.ini ] ---------
..

- [ File = ./tests/pix.ini ] ---------
......................

- [ File = ./tests/huawei_usg.ini ] ---------
...

- [ File = ./tests/checkpoint_smart1.ini ] ---------
..................

- [ File = ./tests/junos.ini ] ---------
...

- [ File = ./tests/web_appsec.ini ] ---------
...............................

- [ File = ./tests/cimserver.ini ] ---------
..

- [ File = ./tests/cisco_ftd.ini ] ---------
..........................................

- [ File = ./tests/firewalld.ini ] ---------
..

- [ File = ./tests/audit_recon.ini ] ---------
..

- [ File = ./tests/openldap.ini ] ---------
.........

- [ File = ./tests/sophos_fw.ini ] ---------
..........

- [ File = ./tests/owlh.ini ] ---------
....

- [ File = ./tests/freepbx.ini ] ---------
......

- [ File = ./tests/syslog.ini ] ---------
......

- [ File = ./tests/openvpn_ldap.ini ] ---------
..

- [ File = ./tests/sophos.ini ] ---------
........

- [ File = ./tests/unbound.ini ] ---------


- [ File = ./tests/github.ini ] ---------
....................................................................................................................................................................................................................................................................................................................................

- [ File = ./tests/cpanel.ini ] ---------
.......

- [ File = ./tests/fireeye.ini ] ---------
...

- [ File = ./tests/fortiauth.ini ] ---------
....

- [ File = ./tests/iptables.ini ] ---------
........

- [ File = ./tests/named.ini ] ---------
.....

- [ File = ./tests/sophos-utm-firewall.ini ] ---------
......

- [ File = ./tests/nginx.ini ] ---------
............

- [ File = ./tests/exim.ini ] ---------
.......

- [ File = ./tests/opensmtpd.ini ] ---------
.......

- [ File = ./tests/oscap.ini ] ---------
................................

- [ File = ./tests/gitlab.ini ] ---------
.........................

- [ File = ./tests/paloalto.ini ] ---------
................

- [ File = ./tests/api.ini ] ---------
............................

- [ File = ./tests/aws_s3_access.ini ] ---------
.......

- [ File = ./tests/exchange.ini ] ---------
..

- [ File = ./tests/f5_big_ip.ini ] ---------
...........................................

- [ File = ./tests/cisco_asa.ini ] ---------
........................................................................................

- [ File = ./tests/nextcloud.ini ] ---------
.......

- [ File = ./tests/trendmicro-cloud-one.ini ] ---------
.................

- [ File = ./tests/pfsense.ini ] ---------
..

- [ File = ./tests/vsftpd.ini ] ---------
....

- [ File = ./tests/web_rules.ini ] ---------
..........

- [ File = ./tests/apache.ini ] ---------
..................

- [ File = ./tests/eset.ini ] ---------
........

- [ File = ./tests/doas.ini ] ---------
....

- [ File = ./tests/sysmon.ini ] ---------
...

- [ File = ./tests/gcp.ini ] ---------
...........

- [ File = ./tests/icinga.ini ] ---------
....

- [ File = ./tests/postfix.ini ] ---------
..

- [ File = ./tests/audit_scp.ini ] ---------
.

- [ File = ./tests/dropbear.ini ] ---------
...

- [ File = ./tests/rsh.ini ] ---------
..

- [ File = ./tests/kernel_usb.ini ] ---------
......

- [ File = ./tests/mcafee_epo.ini ] ---------
.

- [ File = ./tests/netscreen.ini ] ---------
....

- [ File = ./tests/panda_paps.ini ] ---------
........

- [ File = ./tests/powershell.ini ] ---------
...

- [ File = ./tests/auditd.ini ] ---------
...

- [ File = ./tests/cylance.ini ] ---------
.......

- [ File = ./tests/modsecurity.ini ] ---------
......

- [ File = ./tests/cloudlfare-waf.ini ] ---------
.............

- [ File = ./tests/office365.ini ] ---------
................................................................................................................................

- [ File = ./tests/sshd.ini ] ---------
...........................................

- [ File = ./tests/cisco_ios.ini ] ---------
.................

- [ File = ./tests/glpi.ini ] ---------
...

- [ File = ./tests/sudo.ini ] ---------
........

- [ File = ./tests/su.ini ] ---------
.....

- [ File = ./tests/proftpd.ini ] ---------
.......

- [ File = ./tests/SonicWall.ini ] ---------
........

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |   985    |   4174   |  23.60%  |
| Decoders |    76    |   172    |  44.19%  |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/samba.ini        |    4     |    0     |    ✅     |
|./tests/fortigate.ini    |    40    |    0     |    ✅     |
|./tests/apparmor.ini     |    5     |    0     |    ✅     |
|./tests/pam.ini          |    5     |    0     |    ✅     |
|./tests/audit_lateral.ini|    6     |    0     |    ✅     |
|./tests/systemd.ini      |    2     |    0     |    ✅     |
|./tests/arbor.ini        |    2     |    0     |    ✅     |
|./tests/dovecot.ini      |    15    |    0     |    ✅     |
|./tests/mailscanner.ini  |    1     |    0     |    ✅     |
|./tests/ossec.ini        |    5     |    0     |    ✅     |
|./tests/vuln_detector.ini|    2     |    0     |    ✅     |
|./tests/squid_rules.ini  |    2     |    0     |    ✅     |
|./tests/pix.ini          |    22    |    0     |    ✅     |
|./tests/huawei_usg.ini   |    3     |    0     |    ✅     |
|./tests/checkpoint_smart1.ini|    18    |    0     |    ✅     |
|./tests/junos.ini        |    3     |    0     |    ✅     |
|./tests/web_appsec.ini   |    31    |    0     |    ✅     |
|./tests/cimserver.ini    |    2     |    0     |    ✅     |
|./tests/cisco_ftd.ini    |    42    |    0     |    ✅     |
|./tests/firewalld.ini    |    2     |    0     |    ✅     |
|./tests/audit_recon.ini  |    2     |    0     |    ✅     |
|./tests/openldap.ini     |    9     |    0     |    ✅     |
|./tests/sophos_fw.ini    |    10    |    0     |    ✅     |
|./tests/owlh.ini         |    4     |    0     |    ✅     |
|./tests/freepbx.ini      |    6     |    0     |    ✅     |
|./tests/syslog.ini       |    6     |    0     |    ✅     |
|./tests/openvpn_ldap.ini |    2     |    0     |    ✅     |
|./tests/sophos.ini       |    8     |    0     |    ✅     |
|./tests/unbound.ini      |    0     |    0     |    ✅     |
|./tests/github.ini       |   324    |    0     |    ✅     |
|./tests/cpanel.ini       |    7     |    0     |    ✅     |
|./tests/fireeye.ini      |    3     |    0     |    ✅     |
|./tests/fortiauth.ini    |    4     |    0     |    ✅     |
|./tests/iptables.ini     |    8     |    0     |    ✅     |
|./tests/named.ini        |    5     |    0     |    ✅     |
|./tests/sophos-utm-firewall.ini|    6     |    0     |    ✅     |
|./tests/nginx.ini        |    12    |    0     |    ✅     |
|./tests/exim.ini         |    7     |    0     |    ✅     |
|./tests/opensmtpd.ini    |    7     |    0     |    ✅     |
|./tests/oscap.ini        |    32    |    0     |    ✅     |
|./tests/gitlab.ini       |    25    |    0     |    ✅     |
|./tests/paloalto.ini     |    16    |    0     |    ✅     |
|./tests/api.ini          |    28    |    0     |    ✅     |
|./tests/aws_s3_access.ini|    7     |    0     |    ✅     |
|./tests/exchange.ini     |    2     |    0     |    ✅     |
|./tests/f5_big_ip.ini    |    43    |    0     |    ✅     |
|./tests/cisco_asa.ini    |    88    |    0     |    ✅     |
|./tests/nextcloud.ini    |    7     |    0     |    ✅     |
|./tests/trendmicro-cloud-one.ini|    17    |    0     |    ✅     |
|./tests/pfsense.ini      |    2     |    0     |    ✅     |
|./tests/vsftpd.ini       |    4     |    0     |    ✅     |
|./tests/web_rules.ini    |    10    |    0     |    ✅     |
|./tests/apache.ini       |    18    |    0     |    ✅     |
|./tests/eset.ini         |    8     |    0     |    ✅     |
|./tests/doas.ini         |    4     |    0     |    ✅     |
|./tests/sysmon.ini       |    3     |    0     |    ✅     |
|./tests/gcp.ini          |    11    |    0     |    ✅     |
|./tests/icinga.ini       |    4     |    0     |    ✅     |
|./tests/postfix.ini      |    2     |    0     |    ✅     |
|./tests/audit_scp.ini    |    1     |    0     |    ✅     |
|./tests/dropbear.ini     |    3     |    0     |    ✅     |
|./tests/rsh.ini          |    2     |    0     |    ✅     |
|./tests/kernel_usb.ini   |    6     |    0     |    ✅     |
|./tests/mcafee_epo.ini   |    1     |    0     |    ✅     |
|./tests/netscreen.ini    |    4     |    0     |    ✅     |
|./tests/panda_paps.ini   |    8     |    0     |    ✅     |
|./tests/powershell.ini   |    3     |    0     |    ✅     |
|./tests/auditd.ini       |    3     |    0     |    ✅     |
|./tests/cylance.ini      |    7     |    0     |    ✅     |
|./tests/modsecurity.ini  |    6     |    0     |    ✅     |
|./tests/cloudlfare-waf.ini|    13    |    0     |    ✅     |
|./tests/office365.ini    |   128    |    0     |    ✅     |
|./tests/sshd.ini         |    43    |    0     |    ✅     |
|./tests/cisco_ios.ini    |    17    |    0     |    ✅     |
|./tests/glpi.ini         |    3     |    0     |    ✅     |
|./tests/sudo.ini         |    8     |    0     |    ✅     |
|./tests/su.ini           |    5     |    0     |    ✅     |
|./tests/proftpd.ini      |    7     |    0     |    ✅     |
|./tests/SonicWall.ini    |    8     |    0     |    ✅     |
Restarting wazuh-manager...

Restarting wazuh-manager...
- [ File = ./tests/powershell.ini ] ---------
...

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    3     |   4174   |  0.07%   |
| Decoders |    1     |   172    |  0.58%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/powershell.ini   |    3     |    0     |    ✅     |
Restarting wazuh-manager...
  ```
</details>

- [x] 4.d New CDB lists must include the proper test entry in the correct .ini file.

### E2E testing

- [x] 5.a Logs for new or modified decoder/rules sent to a manager running with this PR ruleset appear in Kibana. 

image

- [x] 5.b There are not affected or broken visualizations on Kibana.

image

- [x] 5.c New or modified items can be seen correctly using APP ruleset navigation.

image

### Elasticsearch Template

- [x] 6.a Known fields with output format managed and usually used for searching are included in template array index.query.default_field. 
- [x] 6.b The new field with the correct date format is stored as a "date" type field in the template.
- [x] 6.c The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template. 
- [x] 6.d Known fields with output format managed and usually used for searching are included in the template.

### Stoppers

- [x] 7.a No previous rule ID changes without triple check.
- [x] 7.b No previous decoder name changes without triple check.
- [x] 7.c No previous file name changes without triple check.
- [x] 7.d No previous test changes its 'rule' field value without triple check.

### Others

- [x] 8.a Each file has the correct copyright block.
- [x] 8.b The copyright block doesn't have "Author" only "Created by Wazuh". To include an "Author" request triple check.
- [x] 8.c The copyright block doesn't use "-". 
- [x] 8.d The rule files don't have any sample log.
- [x] 8.e The decoder file has sample logs next to the decoder that matches that log.
- [x] 8.f The decoder and rule files have information about software, version, and any helpful information.
- [x] 8.g The PR has a single commit with CHANGELOG changes in the correct format.
- [x] 8.h The new rule ID is not in use.
- [x] 8.i The new rule ID must be in the defined IDs range. 
- [x] 8.j New rules ID range must be verified with a triple check and noted in the rules ID document.
- [x] 8.k The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template.

### CODE REVIEW

**Review by 72nomada — COMMENTED** [review]
Missing compliance tags in the rules file?

**Review by fabamatic — APPROVED** [review]
LGTM

File: ruleset/testing/tests/powershell.ini [hunk 1]
```diff
@@ -0,0 +1,25 @@
+;  Copyright (C) 2015-2021, Wazuh Inc.
+;
+;  Tests for products: 
+;    Powershell
+;
+;  Sample logs source: 
+;    Powerhsell logs: member.
```
Comment (xrisbarney) [inline ruleset/testing/tests/powershell.ini:7]: Powershell

**Review by xrisbarney — CHANGES_REQUESTED** [review]

**Review by xrisbarney — APPROVED** [review]
LGTM

**Review by 72nomada — APPROVED** [review]
LGTM

**Comment by JcabreraC** [comment]
## Test
```bash
# /var/ossec/bin/wazuh-logtest
Starting wazuh-logtest v4.3.0
Type one log per line

{"win":{"eventdata":{"path":"C:\\\\Users\\\\AtomicRed\\\\AppData\\\\Roaming\\\\TransbaseOdbcDriver\\\\screenshot__.ps1","messageNumber":"1","messageTotal":"1","scriptBlockText":"function screenshot([Drawing.Rectangle]$bounds, $path){ $bmp = New-Object System.Drawing.Bitmap($bounds.width, $bounds.height) $graphics = [Drawing.Graphics]::FromImage($bmp) $graphics.CopyFromScreen($bounds.Location, [Drawing.Point]::Empty, $bounds.size) $bmp.Save($path) $graphics.Dispose() $bmp.Dispose() }","scriptBlockId":"1b79ef82-0efe-4734-b092-3a4b9f17e080"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{a0c1853b-5c40-4b15-8766-3cf1c58f985a}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nfunction screenshot([Drawing.Rectangle]$bounds, $path){ \n $bmp = New-Object System.Drawing.Bitmap($bounds.width, $bounds.height)\n $graphics = [Drawing.Graphics]::FromImage($bmp)\n $graphics.CopyFromScreen($bounds.Location, [Drawing.Point]::Empty, $bounds.size)\n $bmp.Save($path)\n $graphics.Dispose()\n $bmp.Dispose()\n}\r\n\r\nScriptBlock ID: 1b79ef82-0efe-4734-b092-3a4b9f17e080\r\nPath: C:\\Users\\AtomicRed\\AppData\\Roaming\\TransbaseOdbcDriver\\screenshot__.ps1\"","version":"1","systemTime":"2021-06-17T19:42:48.3171903Z","eventRecordID":"95916","threadID":"5292","computer":"hrmanager.ExchangeTest.com","task":"2","processID":"1756","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}}

**Phase 1: Completed pre-decoding.

**Phase 2: Completed decoding.
	name: 'json'
	win.eventdata.messageNumber: '1'
	win.eventdata.messageTotal: '1'
	win.eventdata.path: 'C:\\Users\\AtomicRed\\AppData\\Roaming\\TransbaseOdbcDriver\\screenshot__.ps1'
	win.eventdata.scriptBlockId: '1b79ef82-0efe-4734-b092-3a4b9f17e080'
	win.eventdata.scriptBlockText: 'function screenshot([Drawing.Rectangle]$bounds, $path){ $bmp = New-Object System.Drawing.Bitmap($bounds.width, $bounds.height) $graphics = [Drawing.Graphics]::FromImage($bmp) $graphics.CopyFromScreen($bounds.Location, [Drawing.Point]::Empty, $bounds.size) $bmp.Save($path) $graphics.Dispose() $bmp.Dispose() }'
	win.system.channel: 'Microsoft-Windows-PowerShell/Operational'
	win.system.computer: 'hrmanager.ExchangeTest.com'
	win.system.eventID: '4104'
	win.system.eventRecordID: '95916'
	win.system.keywords: '0x0'
	win.system.level: '5'
	win.system.message: '"Creating Scriptblock text (1 of 1):
function screenshot([Drawing.Rectangle]$bounds, $path){ 
 $bmp = New-Object System.Drawing.Bitmap($bounds.width, $bounds.height)
 $graphics = [Drawing.Graphics]::FromImage($bmp)
 $graphics.CopyFromScreen($bounds.Location, [Drawing.Point]::Empty, $bounds.size)
 $bmp.Save($path)
 $graphics.Dispose()
 $bmp.Dispose()
}

ScriptBlock ID: 1b79ef82-0efe-4734-b092-3a4b9f17e080
Path: C:\Users\AtomicRed\AppData\Roaming\TransbaseOdbcDriver\screenshot__.ps1"'
	win.system.opcode: '15'
	win.system.processID: '1756'
	win.system.providerGuid: '{a0c1853b-5c40-4b15-8766-3cf1c58f985a}'
	win.system.providerName: 'Microsoft-Windows-PowerShell'
	win.system.severityValue: 'VERBOSE'
	win.system.systemTime: '2021-06-17T19:42:48.3171903Z'
	win.system.task: '2'
	win.system.threadID: '5292'
	win.system.version: '1'

**Phase 3: Completed filtering (rules).
	id: '91803'
	level: '14'
	description: 'Screen capture method invoked from PowerShell script.'
	groups: '['windows', ' powershell']'
	firedtimes: '1'
	mail: 'True'
	mitre.id: '['T1113']'
	mitre.tactic: '['Collection']'
	mitre.technique: '['Screen Capture']'
**Alert to be generated.


{"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-ADComputer -Identity 'cfo' -Properties *","scriptBlockId":"f6ea30ec-ff1e-4749-b7d8-0c2003a0c250"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-ADComputer -Identity 'cfo' -Properties *\r\n\r\nScriptBlock ID: f6ea30ec-ff1e-4749-b7d8-0c2003a0c250\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:29:22.450624400Z","eventRecordID":"5347","threadID":"920","computer":"bankdc.ExchangeTest.com","task":"2","processID":"6320","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}}

**Phase 1: Completed pre-decoding.
	full event: '{"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-ADComputer -Identity 'cfo' -Properties *","scriptBlockId":"f6ea30ec-ff1e-4749-b7d8-0c2003a0c250"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-ADComputer -Identity 'cfo' -Properties *\r\n\r\nScriptBlock ID: f6ea30ec-ff1e-4749-b7d8-0c2003a0c250\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:29:22.450624400Z","eventRecordID":"5347","threadID":"920","computer":"bankdc.ExchangeTest.com","task":"2","processID":"6320","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}}'

**Phase 2: Completed decoding.
	name: 'json'
	win.eventdata.messageNumber: '1'
	win.eventdata.messageTotal: '1'
	win.eventdata.scriptBlockId: 'f6ea30ec-ff1e-4749-b7d8-0c2003a0c250'
	win.eventdata.scriptBlockText: 'Get-ADComputer -Identity 'cfo' -Properties *'
	win.system.channel: 'Microsoft-Windows-PowerShell/Operational'
	win.system.computer: 'bankdc.ExchangeTest.com'
	win.system.eventID: '4104'
	win.system.eventRecordID: '5347'
	win.system.keywords: '0x0'
	win.system.level: '5'
	win.system.message: '"Creating Scriptblock text (1 of 1):
Get-ADComputer -Identity 'cfo' -Properties *

ScriptBlock ID: f6ea30ec-ff1e-4749-b7d8-0c2003a0c250
Path: "'
	win.system.opcode: '15'
	win.system.processID: '6320'
	win.system.providerGuid: '{A0C1853B-5C40-4B15-8766-3CF1C58F985A}'
	win.system.providerName: 'Microsoft-Windows-PowerShell'
	win.system.severityValue: 'VERBOSE'
	win.system.systemTime: '2021-06-30T22:29:22.450624400Z'
	win.system.task: '2'
	win.system.threadID: '920'
	win.system.version: '1'

**Phase 3: Completed filtering (rules).
	id: '91805'
	level: '3'
	description: 'Executed Powershell script "Get-ADComputer" executed.'
	groups: '['windows', ' powershell']'
	firedtimes: '1'
	mail: 'False'
	mitre.id: '['T1018']'
	mitre.tactic: '['Discovery']'
	mitre.technique: '['Remote System Discovery']'
**Alert to be generated.


{"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-NetUser","scriptBlockId":"b8633c48-c1ea-47b5-bad6-7766ff636abb"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-NetUser\r\n\r\nScriptBlock ID: b8633c48-c1ea-47b5-bad6-7766ff636abb\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:20:57.104893900Z","eventRecordID":"5329","threadID":"9440","computer":"bankdc.ExchangeTest.com","task":"2","processID":"3940","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}}

**Phase 1: Completed pre-decoding.
	full event: '{"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-NetUser","scriptBlockId":"b8633c48-c1ea-47b5-bad6-7766ff636abb"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-NetUser\r\n\r\nScriptBlock ID: b8633c48-c1ea-47b5-bad6-7766ff636abb\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:20:57.104893900Z","eventRecordID":"5329","threadID":"9440","computer":"bankdc.ExchangeTest.com","task":"2","processID":"3940","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}}'

**Phase 2: Completed decoding.
	name: 'json'
	win.eventdata.messageNumber: '1'
	win.eventdata.messageTotal: '1'
	win.eventdata.scriptBlockId: 'b8633c48-c1ea-47b5-bad6-7766ff636abb'
	win.eventdata.scriptBlockText: 'Get-NetUser'
	win.system.channel: 'Microsoft-Windows-PowerShell/Operational'
	win.system.computer: 'bankdc.ExchangeTest.com'
	win.system.eventID: '4104'
	win.system.eventRecordID: '5329'
	win.system.keywords: '0x0'
	win.system.level: '5'
	win.system.message: '"Creating Scriptblock text (1 of 1):
Get-NetUser

ScriptBlock ID: b8633c48-c1ea-47b5-bad6-7766ff636abb
Path: "'
	win.system.opcode: '15'
	win.system.processID: '3940'
	win.system.providerGuid: '{A0C1853B-5C40-4B15-8766-3CF1C58F985A}'
	win.system.providerName: 'Microsoft-Windows-PowerShell'
	win.system.severityValue: 'VERBOSE'
	win.system.systemTime: '2021-06-30T22:20:57.104893900Z'
	win.system.task: '2'
	win.system.threadID: '9440'
	win.system.version: '1'

**Phase 3: Completed filtering (rules).
	id: '91806'
	level: '3'
	description: 'Powershell script "Get-NetUser executed".'
	groups: '['windows', ' powershell']'
	firedtimes: '1'
	mail: 'False'
	mitre.id: '['T1087.002']'
	mitre.tactic: '['Discovery']'
	mitre.technique: '['Domain Account']'
**Alert to be generated.
```

**Review by JcabreraC — APPROVED** [review]
LGTM !

**Pull request merged**